frontend-react/src/components/FileHandlers/FileHandlerMessaging.tsx

```diff
@@ -220,7 +220,13 @@ const ErrorRow = ({ error, index }: ErrorRowProps) => {
     const { message, field, errorCode, trackingIds } = error;
     return (
         <tr key={"error_" + index}>
-            <td>{errorCode ? ErrorCodeTranslation[errorCode] : message}</td>
+            <td>
+                {errorCode &&
+                errorCode !== "UNKNOWN" &&
+                ErrorCodeTranslation[errorCode]
```

Comment:
Added a third condition here to make sure we don't display empty and fall back on the back end generated message if Experience did not create a mapping for a particular error code. Experience should still create a ticket to improve this code to pull from a localization file instead of hardcoding the language in in ErrorCodeTranslation.

Reply:
Thanks for this!

And I realize this sounds like a copout on our side, but localization isn't a high priority for us at this point, and it'll be a much larger endeavor site-wide.  I definitely agree that pulling it in from a localization file would be the right way to go about it, but since we don't have any i18n scaffolding, I'm wary of introducing it specifically to accommodate this change.  But if/when we actually do tackle localization, we'll absolutely factor in these error code messages!

Reply:
As far as I'm aware, ReportStream's audiences are entirely US based and only used within the US. Is there a requirement at all for localization?  I have not heard of a use case for it.

Reply:
I've heard inklings of it being used in international contexts. I'll check my notes for where I might've heard that.